docs/deployments.adoc

```diff
@@ -0,0 +1,185 @@
+:toc: left
+:toclevels: 3
+:sectanchors: true
+:sectids: true
+:source-highlighter: rouge
+:icons: font
+
+= Deployments and contracts addresses
```

Comment:
Maybe we shouldn't document all these here but in docs.threshold.network? https://docs.threshold.network/extras/contract-addresses

Reply:
Yup, I moved the information to https://docs.threshold.network/extras/contract-addresses in https://github.com/threshold-network/threshold/commit/49395d6e3fe79e15647d78beeafaa3b0429971fe. I'll update the `keep-core` docs to link to that document.

Reply:
Updated, see d4248f76b01ec41955ce5aa1427f33c16afc935f.